python/paddle/fluid/parallel_executor.py

```diff
@@ -279,7 +279,7 @@ def run(self, fetch_list, feed=None, feed_dict=None, return_numpy=True):
                 res.append(res_dict)
             self.executor.feed_tensors_into_local_scopes(res)
 
-        fetch_var_name = '@FETCHED_VAR_NAME@'
+        fetch_var_name = 'fetch'
```

Comment:
I think `fetch_var_name` should be special flag and it doesn't conflict with the var's name of the user's program.
So we should check whether the program have a var that the name is exactly `fetch`.

Reply:
we make them consistent with Executor first. Then we can consider special name